Providers/nxOMSAutomationWorker/MSFT_nxOMSAutomationWorkerResource.schema.mof

```diff
@@ -1,7 +1,5 @@
-[ClassVersion("1.2.0")]
+[ClassVersion("1.0.0")]
```

Comment:
Please test this thoroughly, as it may be seen as a disallowed "downgrade" in versions somewhere.